### PR TITLE
Portfolio

### PR DESCRIPTION
**Pull request by TahenyBELGUITH**
In this part, I create the other sections where:
I use grid display
I use flex display

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,61 +11,238 @@
...
+                        <li class="html"><a href="#">Html</a></li>
+                        <li class="css"><a href="#">CSS</a></li>
+                        <li class="js"><a href="#">javaScript></a></li>
+                    </ul>
+                    <button class="sub-btn" type="button">See projects</button>
+                </div>
+            </section>
+        </main>
+
+
```
Comment (rloterh) [inline index.html:181]: - [ ] There're extra lines after the above section. For a cleaner code, we need not add an extra blank which serves no purpose. Please let's ensure to remove **all** extra lines in the code which are serving no purpose.

File: style.css [hunk 2]
```diff
@@ -6,98 +6,318 @@
...
+  background-color: #e5e5e5;
 }
 
-.wrapper-content {
+.main-header {
   display: flex;
   flex-direction: column;
-  justify-content: center;
+  padding: 48px 24px 136px 24px;
+  background-image: url(./images/bg.svg);
```
Comment (rloterh) [inline style.css:47]: - [ ] Your background image is covering both the `header` section as well as the `projects` section. We actually want this background image to only cover the `project ` section, and our design to follow the provided [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as close possible.

**Your design:**
image

**The original design:**
image

Let's improve this aspect by ensuring that the background image covers only the `project` section.

- [ ] Again, to ensure that our design closely matches with the Figma template design let's ensure that our design matches with the font size, font-weight, margins, and paddings as defined in the original design.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Juliana, 

Good job so far!👏 👏 
There are however some issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- No linter errors


## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -11,61 +11,238 @@
...
+                    <li><a href="#"><img class="image3" src="./images/medium.svg" alt="medium"></a></li>
+                    <li><a href="#"><img class="image5" src="./images/github.svg" alt="github"></a></li>
+                    <li><a href="#"><img class="image6" src="./images/angellist.svg" alt="angellist"></a></li>
+                </ul>
+            </div>
+        </header>
+        <!-- end of wrapper home page -->
+
+        <!-- start of tonic one section -->
+        <main>
```
[the remark this answers is not in the document]
Comment (TahenyBELGUITH) [reply index.html:67]: thank you for your review,
in the first part we have to use the flex display

File: index.html [hunk 4]
```diff
@@ -11,61 +11,238 @@
...
-                <img src="images/logo.png" alt="logo">
-            </a>
-        </div>
-        <div class="toggle">
-            <a href="#">
-                <img src="images/menu.png" alt="menu">
-            </a>
-        </div>
-    </div>
+                <!-- liste social media  -->
```
[the remark this answers is not in the document]
Comment (TahenyBELGUITH) [reply index.html:54]: should I delete all the comments!! I don't understand

File: index.html [hunk 5]
```diff
@@ -11,61 +11,238 @@
...
+                    <li><a href="#"><img class="image6" src="./images/angellist.svg" alt="angellist"></a></li>
+                </ul>
+            </div>
+        </header>
+        <!-- end of wrapper home page -->
+
+        <!-- start of tonic one section -->
+        <main>
+            <section class="tonic-section wrapper-part-content">
+                <div class="tonic-pic">
```
[the remark this answers is not in the document]
Comment (TahenyBELGUITH) [reply index.html:69]: I do this task already  i use list for the projects

File: index.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (rloterh) [reply index.html:54]: Yes, Kindly remove **all** comments.

[the remark this answers is not in the document]
Comment (TahenyBELGUITH) [reply index.html:54]: done

File: index.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (TahenyBELGUITH) [reply index.html:67]: done

File: index.html [hunk 6]
```diff
@@ -11,61 +11,238 @@
...
+                    <img src="images/tonic.svg" alt="tonic part">
+                </div>
+
+                <div class="tonic-info">
+                    <h2 class="tonic-header">Tonic</h2>
+                    <ul class="tonic-tech">
+                        <li class="canopy">CANOPY</li>
+                        <li><span class="counter"><img src="images/counter.png" alt=""></span></li>
```
[the remark this answers is not in the document]
Comment (TahenyBELGUITH) [reply index.html]: done

File: style.css (hunk 2 above)
Comment (TahenyBELGUITH) [reply style.css:48]: done

File: index.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (Emmyn5600) [reply index.html:54]: Nice to implement this.

File: index.html [hunk 7]
```diff
@@ -11,61 +11,247 @@
...
+                        sign-ups
+                        required.</p>
+
+                    <ul class="technologies">
+                        <li class="html"><a href="#">Html</a></li>
+                        <li class="css"><a href="#">CSS</a></li>
+                        <li class="js"><a href="#">javaScript></a></li>
+                    </ul>
+                    <button class="sub-btn" type="button">See projects</button>
+                </div>
```
Comment (Emmyn5600) [inline index.html:148]: - [ ]  Optional but I recommend to do it.
    * The button (e.g., "See project") is a link a or button tag, Nice to implement that however, You need to make the button styling: Active, Hover and Pressed

File: index.html [hunk 8]
```diff
@@ -11,61 +11,247 @@
...
```
Comment (Emmyn5600) [inline index.html:177]: - [ ]  Optional but I recommend to do it.
    * The button (e.g., "See project") is a link a or button tag, Nice to implement that however, You need to make the button styling: Active, Hover and Pressed

File: index.html [hunk 9]
```diff
@@ -11,61 +11,247 @@
...
+                <p class="connect"><a href="#">LET’S CONNECT!</a></p>
+                <!-- liste social media  -->
+                <ul class="social-media">
+                    <li><a href="#"><img class="image1" src="./images/twitter.svg" alt="twitter"></a></li>
+                    <li><a href="#"><img class="image2" src="./images/linkedin.svg" alt="linkedin"></a></li>
+                    <li><a href="#"><img class="image3" src="./images/medium.svg" alt="medium"></a></li>
+                    <li><a href="#"><img class="image5" src="./images/github.svg" alt="github"></a></li>
+                    <li><a href="#"><img class="image6" src="./images/angellist.svg" alt="angellist"></a></li>
+                </ul>
+                <button class="sub-btn about-btn" type="button">Get my Resume</button>
```
Comment (Emmyn5600) [inline index.html:205]: - [ ]  Optional but I recommend to do it.
    * The button (e.g., "Get my Resume") is a link a or button tag, Nice to implement that however, You need to make the button styling: Active, Hover and Pressed

File: index.html [hunk 10]
```diff
@@ -11,61 +11,247 @@
...
+                that needs coding. <br> I’d love to help with it
+            </p>
+        </div>
+        <form>
+            <div class="formula">
+                <input type="text" value="Taheny BELGUITH">
+                <input type="email" value="taheny.belguith@gmail.com">
+                <textarea cols="30" rows="10">Write your message here</textarea>
+                <button  class="sub-btn-formula about-btn" type="button">Get It Touch</button>
+            </div>
```
Comment (Emmyn5600) [inline index.html:252]: - [ ]  Optional but I recommend to do it.
    * The button (e.g., "Get It Touch") is a link a or button tag, Nice to implement that however, You need to make the button styling: Active, Hover and Pressed

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
### Hi @TahenyBELGUITH 

Great work on making the changes requested by the previous reviewer 👏🏽 👏🏽
You've done well with these implementations, but there is some issue to address before approval. Not to worry, you're almost there! 👍🏽

##  Required Changes :recycle:

**_Check the comments under the review._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 7 above)
Comment (TahenyBELGUITH) [reply index.html:148]: Done

File: index.html (hunk 10 above)
Comment (TahenyBELGUITH) [reply index.html:252]: All are done thanks for your review

**Review by hemant-soni-vst-au4 — APPROVED** [review]
Hi @TahenyBELGUITH, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] Kindly add a little more width to the image in each project so that it looks like Figma design.


Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**